modules/external-api/src/main/java/org/opencastproject/external/common/ApiVersion.java

```diff
@@ -24,7 +24,8 @@
 public enum ApiVersion {
 
   VERSION_1_0_0(1, 0, 0),
-  VERSION_1_1_0(1, 1, 0);
+  VERSION_1_1_0(1, 1, 0),
+  VERSION_1_2_0(1, 2, 0);
 
   /** The most recent version of the External API */
   public static final ApiVersion CURRENT_VERSION = VERSION_1_1_0;
```

Comment:
 @KatrinIhler If the comment is correct, you should change CURRENT_VERSION to VERSION_1_2_0

Reply:
You're right, fixed!